Source/santactl/Commands/SNTCommandRule.m

```diff
@@ -171,6 +213,13 @@ - (void)runWithArguments:(NSArray *)arguments {
       MOLCodesignChecker *cs = [fi codesignCheckerWithError:NULL];
       newRule.identifier = cs.leafCertificate.SHA256;
     } else if (newRule.type == SNTRuleTypeTeamID) {
+      // Get the team ID from the binary.
+      MOLCodesignChecker *cs = [fi codesignCheckerWithError:NULL];
+      if (![self isWellformedTeamID:cs.signingInformation[@"teamid"]]) {
+        [self printErrorUsageAndExit:
+                @"Invalid Team ID in binary (must be 10 alphanumeric characters (e.g. EQHXZ8M8AV)"];
+      };
```

Comment:
This is an odd check, generally if there isn't a team ID the binary is unsigned or adhoc signed, we should report that as this error could be confusing.